### DISCUSSION TITLE
Pyfolio Graphs improvements

### DISCUSSION THREAD

**Discussion by MBounouar** [stefan-jansen/pyfolio#5] [Ideas]
I believe pyfolio has a lot of room for improvements when it comes to plots to make it more appealing.
I personally don't like the seaborn darkgrid, which can be easily changed without touching the code. However, I find the choice of default colors and line width unpleasant.
I played around with the settings for the graphs. Let me know what you think and maybe suggest. 
I would gladly make a pull request with some fixes and improvements.

![Original pyfolio graph](https://user-images.githubusercontent.com/28890559/130859007-be724788-1e04-45b9-8570-73f183063724.png)
![Example changing default colors and line width](https://user-images.githubusercontent.com/28890559/130859038-1dce657d-4b31-40a9-8b4f-a25a46e87f00.png)

**Comment by stefan-jansen** [comment]
Thank you @MBounouar for the initiative (again)! Very much agree that the current plots are no aesthetic winners and definitely look a bit out of date. Your samples above are already a significant improvement and PRs will of course be most welcome. Given how much has visualization improved since pyfolio got started, perhaps we can (over time) think about adding more user-friendly (interactive) plots.

**Reply by MBounouar** [reply]
You are welcome. Some people like the darkgrid mode but I think it's a waste of toner :-)  color.

For interactive plots I don't really see an alternative to using plotly and either Flask or Dashplotly to get up and going, each with pros and cons.
It would be interesting to see if  someone has a different view on that.

**Comment by trad4r** [comment]
@MBounouar Thanks for your improvement. Can you share your method to modify the Pyfolio graph settings?

**Reply by MBounouar** [reply]
Hi,
What you need is essentially here 

https://github.com/stefan-jansen/pyfolio-reloaded/blob/d7dc3a4d14c7173a63e41926fde34e4f9f23d664/pyfolio/plotting.py#L82

I created a quick and dirty gist to show how I created the example

https://gist.github.com/MBounouar/e135583e33f58da2bab5fd0265e7cc8a

Don't hesitate to give a feedback if you would like to see some changes in the graphs

Now I see that I forgot to mention that there are some changes that a required in the line widths and color to match exactly my "improvements".
